CLAUDE.md

```
@@ -0,0 +1,54 @@
+```bash
+export JAVA_HOME="$(mise where java@$(sed -n 's/^jdkBuildVersion=//p' gradle.properties))"
+```
+
+This needs the `java@<N>` alias to point at an installed version; `mise where` fails otherwise.
+
+## Long-running commands
+
+`./gradlew clean build` takes several minutes — well over the default `Bash` timeout. Pass an
+explicit one:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)